src/mixins/meta.ts

```diff
@@ -61,15 +83,26 @@ export function PageMetaI18nMixin(
         metaI18nTitle: string;
         metaI18nDescription: string;
         metaI18nRoutePath: string;
+        metaI18nAlternateLocale?: QMetaI18nAlternate;
       }
     ) {
+      const alternateLocaleData = this.metaI18nAlternateLocale;
+      // Generate the alternate meta tags.
+      const alternateMetaTags = pageAlternateLocales(alternateLocaleData?.locales ?? [], alternateLocaleData?.currentLocale ?? undefined);
```

Comment:
Here destructuring can be used as well, to provide defaults here, removing 

```ts
const { locales = [], currentLocale } = this.metaI18nAlternateLocale ?? {};

const {  metaAlternate: pageAlternateMetaTags, links: pageAlternateLinks } = pageAlternateLocales(locales, currentLocale);

return {
        meta: {
          ...pageSocialMetaTags(
            this.metaI18nTitle,
            this.metaI18nDescription,
            this.$route.path
          ),
          ...pageAlternateMetaTags,
        },
        link: pageAlternateLinks
      };
```

Reply:
I added `page` to the variable to differentiate them from `layout` ones.
When we reorganize the code in multiple files we can drop this prefix

Reply:
Updated accordingly.